src/main/java/htsjdk/variant/vcf/VCFEncoder.java

```diff
@@ -201,33 +198,66 @@ else if ( val instanceof List ) {
 		return result;
 	}
 
+	private static double round(final double value, final int digits) {
```

Comment:
should this be in some MathUtils class?

Reply:
We had discussed using a MathUtils class for this, but none exists for rounding to a particular number of significant digits.  It's not clear to me that the current code correctly rounds to the number of significant digits as we would like it to.

I think we want a function that does something like:  (for digits = 4)
0.000003599999 -> 0.0000036
4.12345 -> 4.123
1234.345 -> 1234